src/codeu/chat/client/simplegui/landing_frame/CreateAccountPanel.java

```diff
@@ -0,0 +1,42 @@
+package codeu.chat.client.simplegui;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Created by Suveena on 5/18/17.
+ */
+
+@SuppressWarnings("serial")
+public class CreateAccountPanel extends JPanel{
+
+    public CreateAccountPanel() {
+        super(new GridBagLayout());
+        initialize();
+    }
+
+    private void initialize() {
+
+        // This panel contains from top to bottom; a message, a button.
+
+        // Set layout within panel
+        JPanel InnerLayout = new JPanel();
+        InnerLayout.setLayout(new BoxLayout(InnerLayout, BoxLayout.Y_AXIS));
+
+        JLabel userQuestionLabel = new JLabel("Create an Account");
+        JButton createAccountButton = new JButton("Get Started!");
+
+        InnerLayout.add(userQuestionLabel);
+        InnerLayout.add(createAccountButton);
+
+        this.add(InnerLayout);
+
+        /*signInButton.addActionListener(new ActionListener() {
```

Comment:
Remove dead code.  As APIs and libraries change, the commented out code just takes up space and most likely end up deleted eventually anyway.

You can keep commented out code in local/personal branches, but try to keep them from the common repo